### DISCUSSION TITLE
Preferred OS for Machinaris

### DISCUSSION THREAD

**Discussion by olivierLA75** [guydavis/machinaris#139] [Q&A] [answered]
Hi, 
which OS would you recommend for a multi-machine system? I'm on Windows at the moment but I was wondering if the overall performance/reliability would be better on Linux (Ubuntu)
Best,
Olivier

**Comment by guydavis** [comment, marked as answer]
Yes, if the machine is dedicated to Chia and other-Linux friendly services, I would recommend Linux such as Ubuntu.  However, if the machine is used daily for Windows, then Machinaris is a good option as you can run the bundled Linux-versions of Chia, Plotman, Madmax within the Machinaris Docker container.  However, running on Windows in Docker will definitely be much slower performance-wise for plotting.  Definitely trade-offs to consider.   Hope this helps!

**Comment by olivierLA75** [comment]
Thanks for your quick answer, my machines are dedicated to Chia so I’m going to start switching them to Ubuntu. I don’t know if you saw it but I sent you an email as I’m looking for a little hand to setup Machinaris (I have zero knowledge of Docker and programming). Looking forward to hearing from you. Thanks! Olivier From: Guy Davis ***@***.***>Date: Sunday, July 4, 2021 at 6:33 PMTo: guydavis/machinaris ***@***.***>Cc: olivierLA75 ***@***.***>, Author ***@***.***>Subject: Re: [guydavis/machinaris] Preferred OS for Machinaris (#139)Yes, if the machine is dedicated to Chia and other-Linux friendly services, I would recommend Linux such as Ubuntu. However, if the machine is used daily for Windows, then Machinaris is a good option as you can run the bundled Linux-versions of Chia, Plotman, Madmax within the Machinaris Docker container. However, running on Windows in Docker will definitely be much slower performance-wise for plotting. Definitely trade-offs to consider. Hope this helps!—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Reply by guydavis** [reply]
Just re-read through your response which is mostly HTML as seen above.  Now I see what you actually wrote.  If you're having trouble with Machinaris install, drop into the [Discord](https://discord.gg/mX4AtMTt87) for assistance..
